ui/src/components/ESSelector/ESRelatedSelector.js

```diff
@@ -264,7 +264,9 @@ ESRelatedSelector.propTypes = {
   query: PropTypes.func,
   onSelectResult: PropTypes.func,
   onRemoveSelection: PropTypes.func,
-  renderSelections: PropTypes.func,
+  /* element to render one selected entry */
   renderSelection: PropTypes.func,
+  /* element to render list of selected entries */
+  renderSelections: PropTypes.func,
```

Comment:
Can we use one function and in the case we want to render a single thing just to pass an array with one entry, instead of having 2 functions?